spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+RSpec.describe UsersController, type: :model do 
+	it 'is not valid without a first_name' do
+    user = User.new(last_name: 'last Name')
+    expect(user.valid?).to be_falsey
+  end
+
+  it 'is not valid without a last_name' do
+    user = User.new(first_name: 'Name')
+    expect(user.valid?).to be_falsey
+  end 
+end
```

Comment:
fix the indentation of this file

Reply:
Or you can also write it as below:
```
it { is_expected.to validate_presence_of(:first_name) }
it { is_expected.to validate_presence_of(:last_name) }
```

Also I am missing the enum values specs here